Bring consistency to sample titles (#192)
Esri/arcgis-maps-sdk-unity-samples: sample_project/Assets/SampleViewer/Samples/FeatureLayer/readme.md

2. Input your API key under the **API Key Drop down**.
3. Click the **Sample Drop Down** and select **Feature Layer**.

## How to use the sample

> 1. Already available is a Feature Layer of park trees. Upon running the scene, these feature items will be spawned in at their default latitude and longitude, and with all of the properties associated with that feature item. 

Reviewer: ```suggestion
1. A Feature Layer of park trees is already available. When you run the scene, these feature items are spawned at their default latitude and longitude with all their associated properties. 
```
There's nothing wrong with the original, but here's a proposed rewording to make it a little less formal.

> 2. Clicking on any of these objects under the **FeatureLayer** Gameobject in Unity to see its latitude, longitude, and any properties received by the query.

Reviewer: ```suggestion
2. Click on any of these objects under the **FeatureLayer** Gameobject in Unity to see its latitude, longitude, and any properties received by the query.
```
